package.json

```diff
@@ -1,36 +1,43 @@
 {
     "name": "@prezly/sdk",
-    "version": "1.0.0",
+    "version": "1.0.0-2",
     "description": "Prezly API SDK",
     "main": "dist/index.js",
+    "files": [
```

Comment:
So I'm trying to use this package (yes, `"@prezly/sdk": "^1.0.0-2"`), I have this code:

```ts
import PrezlySdk from '@prezly/sdk';

const prezlySdk = new PrezlySdk({ accessToken: PREZLY_ACCESS_TOKEN });
```
and I get this error:

```bash
src/index.ts:13:27 - error TS2351: This expression is not constructable.
  Type 'typeof import("/home/kamil/projects/prezly/kantar-sync/node_modules/@prezly/sdk/dist/index")' has no construct signatures.

13     const prezlySdk = new PrezlySdk({ accessToken: PREZLY_ACCESS_TOKEN });
```

`node_modules/@prezly/sdk/dist/index.d.ts` contains only this:
```ts
export {};
```

Reply:
Hmm. I was assuming commonjs, i.e. `require(@prezly/sdk)`. I assume the `import` expect the 'module.default' key, which doesn't work. I will change it with the next release.

Reply:
Should be fixed with 8c7c07154f6b1277c07432cd90eb643e628b9473.